allow-popups-to-escape-sandbox/index.html

```diff
@@ -0,0 +1,68 @@
+---
+feature_name: Allow popups to escape sandboxed iframe
+chrome_version: 46
+feature_id: 5708368589094912
+---
+
+<p>Google's anti-malvertising team wants to mitigate the risks associated with
```

Comment:
I'd be concerned about including "Google's _XYZ_ team wants so-and-so" in the samples as justification for a new feature. This feature is going through the W3C process, so whatever the initial motivation was, the spec is going to end up getting feedback from other members of the wider open web community.

Could you rephrase as _"In order to mitigate the risks associated with ads when using sandboxed iframes, developers can now add..."_